### PR TITLE
Sequential Dispatching

### LINKED ISSUE #5: Support for sequential dispatching
Support dispatching the tasks sequentially by building a topological order.

### PR DESCRIPTION
**Pull request by torkleyy**
Fixes #5

### CODE REVIEW

**Comment by vitvakatu** [comment]
Oh, I missed that, sorry. I'll take a look now

**Comment by torkleyy** [comment]
Thanks. Do you know why `--no-default-features` fails? Seems it's a`travis-cargo` problem.

File: src/dispatch.rs [hunk 1]
```diff
@@ -52,27 +52,62 @@ pub struct Dispatcher<'c, 't, C = ()> {
...
+    ///
+    /// This function automatically redirects to
+    ///
+    /// * [`dispatch_par`] in case the `parallel` feature is set
+    /// * [`dispatch_seq`] in case the `parallel` feature is not set
+    ///
+    /// [`dispatch_par`]: struct.Dispatcher.html#method.dispatch_par
+    /// [`dispatch_seq`]: struct.Dispatcher.html#method.dispatch_seq
+    pub fn dispatch(&mut self, res: &mut Resources, context: C) {
+        self.dispatch_cfg(res, context);
```
Comment (vitvakatu) [inline src/dispatch.rs]: Why do you need this helper function? You can use the configuration flags directly here, can't you?

**Comment by torkleyy** [comment]
@vitvakatu Thanks for taking a look!

**Pull request merged**